### PR TITLE
New filter admin_canonical_url

### PR DESCRIPTION
**Pull request by prionkor**
Trac ticket: 

https://core.trac.wordpress.org/ticket/59545

Related PR https://github.com/WordPress/wordpress-develop/pull/1504

Trac ticket: 

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/includes/misc.php [hunk 1]
```diff
@@ -1398,6 +1398,15 @@ function wp_admin_canonical_url() {
 	// Ensure we're using an absolute URL.
 	$current_url  = set_url_scheme( 'http://' . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'] );
 	$filtered_url = remove_query_arg( $removable_query_args, $current_url );
+
+	/**
+	 * Filters the admin canonical url value
```
Comment (audrasjb) [inline src/wp-admin/includes/misc.php]: Missing final period in this sentence :)

File: src/wp-admin/includes/misc.php [hunk 2]
```diff
@@ -1398,6 +1398,15 @@ function wp_admin_canonical_url() {
...
+	 *
+	 * @since 6.3.2
```
Comment (audrasjb) [inline src/wp-admin/includes/misc.php]: We don't introduce any new feature in a minor release, let's target 6.4.0 instead :)

**Review by audrasjb — CHANGES_REQUESTED** [review]
Thanks for the patch! It looks good to me, I added some change requests ;)

File: src/wp-admin/includes/misc.php (hunk 1 above)
Comment (prionkor) [reply src/wp-admin/includes/misc.php]: thank you! Updated PR

File: src/wp-admin/includes/misc.php (hunk 2 above)
Comment (swissspidy) [reply src/wp-admin/includes/misc.php]: I'd say 6.5 as this is an enhancement still.

**Comment by prionkor** [comment]
The failed test is timeout and not relevant to the change. What is the next step?

**Comment by audrasjb** [comment]
> The failed test is timeout and not relevant to the change. What is the next step?

No worries, it clearly appears that it is a timeout, so this is not a blocker.

**Review by audrasjb — APPROVED** [review]

**Comment by audrasjb** [comment]
committed in https://core.trac.wordpress.org/changeset/57595

**Pull request closed without merging**